File: mpp_proiect_java/src/main/java/triatlon/repository/IRezultatRepository.java
package triatlon.repository;

import triatlon.domain.Proba;
import triatlon.domain.Rezultat;

import java.util.List;


public interface IRezultatRepository extends IRepository<Long, Rezultat>{
     List<Rezultat> filterByProba(Proba proba);
}

File: mpp_proiect_CSharp/triatlon/persistence/IRezultatRepository.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using triatlon.domain;

namespace triatlon.repository
{
    public interface IRezultatRepository : IRepository<long, Rezultat>
    {
        IEnumerable<Rezultat> filterByProba(Proba proba);
    }
}